uniprot-config/src/main/resources/search-fields-config/crossref-search-fields.json

```diff
@@ -37,20 +26,37 @@
     "example": "category_str desc"
   },
   {
+    "id": "category_facet",
+    "fieldName": "category_facet",
```

Comment:
if category_facet is just a string field in schema.xml, it could be used as an exact search on that field, right?

If so, would it make sense to not call it category_facet, and maybe category_exact or something?